### DISCUSSION TITLE
feat: support Feishu calendar

### DISCUSSION THREAD

**Discussion by love98ooo** [NJUPT-SAST/sast-evento-backend#91] [Ideas]
To make the SAST workflow more effective and standard, we need to support Feishu calendar.

Relevant: 
[Calendar Overview](https://open.feishu.cn/document/server-docs/calendar-v4/overview)

**Comment by feellmoose** [comment]
需要的功能具体有哪些呢？
例如：
1. 创建活动时创建日程/会议室预定
2. 同步日程到evento

**Reply by love98ooo** [reply]
只需要第二个，而这可能需要飞书权限
